daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
@jhowardmsft the driver technically should work, but the test effort for use to verify on Windows is pretty significant today, based on our recent eval. Is this something that you guys could help us with? We'd be happy to get you a Splunk instance you could test pushing to.